x-pack/filebeat/input/azureeventhub/config.go

```diff
@@ -26,6 +26,8 @@ type azureInputConfig struct {
 	SAContainer string `config:"storage_account_container"`
 	// by default the azure public environment is used, to override, users can provide a specific resource manager endpoint
 	OverrideEnvironment string `config:"resource_manager_endpoint"`
+	// option to cleanup the input for known issues which might produce a malformed JSON
+	SanitizeMessage bool `config:"sanitize_message"`
```

Comment:
Can we offer the option of removing new lines or replacing single quotes as two options instead of just one that does both?

Function App logs users are affected only by the single quotes problem; App Service logs users are affected by the new lines problem only. We can enable only the required adjustment in the settings for each integration to lower the risk of side effects.

WDYT?

Reply:
@zmoog  Sounds good, just added the initial solution for that (still needs docs and maybe slight touch-ups). 

Let me know if this looks like what you had in mind.